collection.css

```diff
@@ -0,0 +1,78 @@
+body {
+    margin: 0;
+    background-color: #DBC8A7;
+}
+
+.guitar {
+    width: 20%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 20px;
+}
+
+.title {
+    color: #180E1A;
+    text-align: center;
+    font-size: 40px;
+    font-family: fantasy;
+}
+
+.collection {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    width: 10%;
+    margin: auto;
+    grid-column-gap: 30px;
+    grid-row-gap: 80px;
+    justify-content: center;
+    margin-top: 45px;
+    font-family: 'Roboto Mono', monospace;
+    padding-top: 30px;
+    text-align: left;
+}
+
+@media only screen and (max-width: 10%) {
```

Comment:
I think the max-width needs to be set as a px width as it may not know what the % is of